flocker/acceptance/endtoend/test_dataset.py

```diff
@@ -102,9 +102,14 @@ def _get_package_source(self, default_version=None):
         ),
     )
     @skip_backend(
-        unsupported={backends.LOOPBACK},
-        reason="Does not maintain compute_instance_id across restarting "
-               "flocker (and didn't as of most recent release).")
+        unsupported={backends.LOOPBACK, backends.OPENSTACK},
+        reason=(
+            "Loopback backend does not maintain compute_instance_id across "
+            "restarting flocker (and didn't as of most recent release). "
+            "OpenStack backend (as of 1.15.0) cannot reliably identify "
+            "the compute node if it has floating IP addresses."
```

Comment:
nit: maybe worth putting a link the the text to where we track the bug? Or to a place where there is more detail to someone who comes long after. up to you :)